dev_guide/integrating_external_services.adoc

```diff
@@ -32,22 +33,22 @@ The solution for integrating with an external database includes:
 2.  One or more `Endpoints` for the service
 3.  Environment variables in the appropriate pods containing the credentials
 
-=== Creating the Service
-
-The first step in integrating with an external service is to create an OpenShift service to
-represent it. This is similar to creating an internal service; the difference is in the
-service's `Selector` field.
-
-Internal OpenShift services use the `Selector` field to associate pods with services
-via labels. A system component called `EndpointsController` synchronizes the endpoints for
-services that specify selectors with the pods that match the selector. The service proxy and
-OpenShift router load-balance requests to the service amongst the service's endpoints.
-
-The only difference between an internal OpenShift service and a service that represents an external
-resource is that there are no pods to associate an external service with. To represent external
-services, we just need to leave the `Selector` field unset. This will make the `EndpointsController`
-ignore the service and allow you to specify endpoints manually:
-
+. Create an OpenShift service to represent your external database. This is
+similar to creating an internal service; the difference is in the service's
+`Selector` field.
++
+Internal OpenShift services use the `Selector` field to associate pods with
+services via labels. A system component called `EndpointsController`
+synchronizes the endpoints for services that specify selectors with the pods
+that match the selector. The service proxy and OpenShift router load-balance
+requests to the service amongst the service's endpoints.
++
+Services that represent an external resource do not require pods to be
+associated with an external service. For external services, leave the `Selector`
+field unset. This represents the external service, making the
+`EndpointsController` ignore the service and allows you to specify endpoints
+manually:
++
 ====
```

Comment:
Also, same as above, but I reworded it a little so it doesn't repeat the same phrase over and over again.